plugins/hls-fourmolu-plugin/src/Ide/Plugin/Fourmolu/Shim.hs

```diff
@@ -0,0 +1,68 @@
+{-# LANGUAGE CPP #-}
+
+module Ide.Plugin.Fourmolu.Shim (
+  -- * FourmoluConfig
+  FourmoluConfig (..),
+  toConfig,
+  emptyConfig,
+
+  -- * FixityMap
+  addFixityOverrides,
+
+  -- * ConfigParseError
+  showParseError,
+) where
+
+import           Ormolu.Config
+
+#if MIN_VERSION_fourmolu(0,7,0)
+import           Ormolu.Fixity
+#endif
+
+{-- Backport FourmoluConfig --}
+
+#if MIN_VERSION_fourmolu(0,7,0)
+toConfig :: FourmoluConfig -> FourmoluConfig
+toConfig = id
+#else
+data FourmoluConfig = FourmoluConfig
+  { cfgFilePrinterOpts :: PrinterOptsPartial
+  , cfgFileFixities    :: FixityMap
+  }
+
+toConfig :: PrinterOptsPartial -> FourmoluConfig
+toConfig opts =
+  FourmoluConfig
+    { cfgFilePrinterOpts = opts
+    , cfgFileFixities = mempty
+    }
+#endif
+
+emptyConfig :: FourmoluConfig
+emptyConfig =
+  FourmoluConfig
+    { cfgFilePrinterOpts = mempty
+    , cfgFileFixities = mempty
+    }
```

Comment:
will eventually be under a `#if !MIN_VERSION_fourmolu(0,8,1)` macro: https://github.com/fourmolu/fourmolu/pull/221